<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calabozo RPG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QProgressBar" name="barraVida">
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLanzarDados">
      <property name="text">
       <string>Lanzar Dados</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAtacar">
      <property name="text">
       <string>Atacar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
